File: Data Structures & Algorithms/final-array-state-after-k-multiplication-operations-i/submission-0.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, mul: int) -> List[int]:
       import heapq as hq

       pq = [(num, i) for i, num in enumerate(nums)]
       hq.heapify(pq)
       print(pq)
       for _ in range(k):
            x, i = hq.heappop(pq)
            nums[i] = x*mul
            hq.heappush(pq, (nums[i], i))

       return nums

File: Data Structures & Algorithms/single-number/submission-0.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        unordered_map<int, int> mp;
        for(auto c: nums){
            if (count(nums.begin() , nums.end(), c)==1) return c;
        }
        return 0;
    }
};
